assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/methylseq/releases/tag/2.5.0" target="_blank">nf-core/methylseq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/methylseq/releases/tag/2.6.0dev" target="_blank">nf-core/methylseq</a>
```

Comment:
I think these need to be `dev`

Reply:
ya i was confused about it.....I saw that fetchngs actually got rid of this file ? any idea why it was not needed there and is needed in this pipeline ?

Reply:
part of the new refactoring i guess ?